File: app/src/main/java/com/BlizzardArmory/model/diablo/item/Secondary.java
package com.BlizzardArmory.model.diablo.item;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


/**
 * The type Secondary.
 */
public class Secondary {

    @SerializedName("text")
    @Expose
    private String text;
    @SerializedName("textHtml")
    @Expose
    private String textHtml;

    /**
     * Gets text.
     *
     * @return the text
     */
    public String getText() {
        return text;
    }

    /**
     * Gets text html.
     *
     * @return the text html
     */
    public String getTextHtml() {
        return textHtml;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/diablo/item/Secondary.kt
package com.BlizzardArmory.model.diablo.item

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Secondary.
 */
@Keep
data class Secondary(

    @SerializedName("text")

    val text: String,

    @SerializedName("textHtml")

    val textHtml: String

)